nin/backend/nin

```diff
@@ -63,6 +64,29 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    echo('Checking for new version of nin');
+
+    var ninExecutable = __dirname;
+    var ninPath = ninExecutable.split('/');
+    var ninPath = ninPath.splice(0, ninPath.length-3).join('/');
```

Comment:
Okay, I'm trying to understand what the value of ninPath is intended to be.
`which('nin')` on my machine didn't return anything, since I had followed to install guide in the readme and only setup an alias in my .bashrc (nin was not in path).

`__dirname` returns "/Users/stiaje/Projects/nin", which I guess might be one less "nin" than what `which` returned.

`ninPath` is then finally set to "/Users" for me.

What is the value of ninPath when you run it?